src/main/java/racing/Simulator.java

```diff
@@ -1,33 +1,58 @@
 package racing;
 
+import racing.model.RacingCar;
+import racing.view.InputView;
+import racing.view.ResultView;
+
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public class Simulator {
     private static final int BEGINNING = 0;
 
+    static String[] carName;
     static int carCount;
     static int raceRound;
     static List<RacingCar> cars = new ArrayList<>();
 
     public static void main(String[] args) {
-        carCount = InputView.getCarCount();
+        carName = InputView.getCarName();
+        carCount = carName.length;
         raceRound = InputView.getRaceRound();
         for (int i = 0; i < carCount; i++) {
-            cars.add(new RacingCar(BEGINNING));
+            cars.add(new RacingCar(BEGINNING, carName[i]));
         }
         System.out.println();
         System.out.println("실행 결과");
         for (int i = 0; i < raceRound; i++) {
             raceAllCars();
             System.out.println();
         }
+        ResultView.showWinners(getWinners(cars));
+    }
+
+    public static List<RacingCar> getWinners(List<RacingCar> cars) {
+        int topSpeed = getTopSpeed(cars);
+        return cars.stream()
+                .filter(car -> car.getProgress() == topSpeed)
+                .collect(Collectors.toList());
+    }
+
+    public static int getTopSpeed(List<RacingCar> cars) {
+        int max = 0;
+        for (RacingCar car : cars) {
+            max = Math.max(max, car.getProgress());
```

Comment:
```suggestion
            max = car.max(max);
```

객체에게 검증하라고 시켜보는관점도 있을것 같아요 🤔

Reply:
질문이 하나 있습니다! 해당 부분은 모든 RacingCar 중 최대 값을 구하는 부분입니다. 해당 부분을 객체가 어떻게 검증할지 감이 잘 안잡히는 것 같습니다.. 혹시 힌트를 조금 더 주실 수 있을까요..?